app/presenters/url.rb

```diff
@@ -0,0 +1,22 @@
+class Url
```

Comment:
since this is a presenter I think that using `LinkHelper` would be confusing (prefer to stay away from Helpers in general since the methods inside of them have a global scope which is 😷 ) 

I think you have a point about the name, though. I will go with `Link` ! Updated at 34108a67